WalletWasabi.Gui/Config.cs

```diff
@@ -149,18 +149,11 @@ public Uri GetCurrentBackendUri()
 				return _backendUri;
 			}
 
-			if (Network == Network.Main)
-			{
-				_backendUri = new Uri(MainNetBackendUriV3);
-			}
-			else if (Network == Network.TestNet)
-			{
-				_backendUri = new Uri(TestNetBackendUriV3);
-			}
-			else // RegTest
-			{
-				_backendUri = new Uri(RegTestBackendUriV3);
-			}
+			_backendUri = Network == Network.Main
+				? new Uri(MainNetBackendUriV3)
+				: Network == Network.TestNet
+					? new Uri(TestNetBackendUriV3)
+					: new Uri(RegTestBackendUriV3);
```

Comment:
I agree and I honestly think if else is easier to read, but I did this in the first place only because conditional expression is used a lot when possible in the code base.